metrics/ci_integrations/lib/source/github_actions/config/validator/github_actions_source_validator.dart

```diff
@@ -117,7 +117,44 @@ class GithubActionsSourceValidator
       );
     }
 
-    return _finalizeValidationResult(GithubActionsStrings.notImplemented);
+    final jobName = config.jobName;
+    final jobNameInteraction = await validationDelegate.validateJobName(
+      workflowId: workflowId,
+      jobName: jobName,
+    );
+
+    if (jobNameInteraction.isSuccess && jobNameInteraction.result == null) {
+      _setUnknownFieldValidationResult(
+        GithubActionsSourceConfigField.jobName,
+        jobNameInteraction.message,
+      );
+    } else {
+      _processInteraction(
+        interaction: jobNameInteraction,
+        field: GithubActionsSourceConfigField.jobName,
+      );
+    }
```

Comment:
It would be great to extract this part of the code into the method to avoid duplication. What do you think?